utils/release/npm-publish-package.mjs

```diff
@@ -180,15 +204,28 @@ function isPrivatePackage() {
   return packageJson.private;
 }
 
+/**
+ *
+ * @param {string} packageName
+ * @param {string} nextGoldVersion
+ * @returns {Promise<string>}
+ */
 async function getNextBetaVersion(packageName, nextGoldVersion) {
+  let nextBetaVersion = `${nextGoldVersion}-0`;
   const registryMeta = await fetchNpm(packageName);
+
+  /**
+   * @type {string[]}
+   */
+  // @ts-ignore force-cast.
```

Comment:
why is this necessary?

Reply:
Because `fetchNpm` is loosely typed and return an `unknown` type.

Reply:
Got it, thanks. Maybe add a brief note as comment ?